sections/introduction.tex

```diff
@@ -1,8 +1,13 @@
 \section{Introduction}\label{sec:introduction}
+% Intro to meta data
+Data is everywhere and it is used in many different scenarios and use cases. 
+When referring to a data set, we talk about data having meta-data information or attributes, which are the entries that could describe certain features about the data, like petal length from well-known IRIS data set\footnote{https://archive.ics.uci.edu/ml/datasets/iris} describing the length of a particular flower's petals.
```

Comment:
```suggestion
When referring to a data set, we talk about a dataset having meta-data information, which is non-essential data features. 
One example of this could be the publication date of a text document. In most contexts, it is not needed to use and understand the document, however it provides more context to the main content of the document: the text.
```

Reply:
```suggestion
When referring to a dataset, we talk about a dataset including metadata information, which are non-essential data features.
One example of this could be the publication date of a text document. 
In most contexts, it is not needed to use and understand the document, however it provides more context to the main content of the document: the text.
```